<launch>
	<node pkg="seabee3_driver_base" type="seabee3_driver_base" name="seabee3_driver_base" />
	<mode pkg="xsens_node" type="xsens_node" name="xsens_node" />
	<node pkg="seabee3_driver_imu" type="seabee3_driver_imu" name="seabee3_driver_imu" />
	
	<!--<include file="$(find seabee3_launch)/teleop_joy.launch" />-->
</launch>
